docs/app/insights.md

```diff
@@ -183,17 +179,47 @@ In order to use this Panel, your Collection will need a datetime Field.
 - **Show X-axis** – Toggles display of time along X-axis.
 - **Show Y-axis** – Toggles display of numeric values along Y-axis.
 
+::: warning Time-oriented metrics only
+
+In order to use this Panel, your Collection will need a datetime Field.
+
+:::
+
 :::tip Custom Date Range
 
 Note that `Past` is not prefixed to a custom range as it is on the default ranges. When selecting **"Other"**, simply
 type in `3 years`, `1 month`, `2 weeks`, `5 days`, etc.
 
 :::
 
-## Extensibility Options
+## Global Variable
+
+![Global Variables](https://cdn.directus.io/docs/v9/app-guide/insights/insights-220624A/global-variable-220624A.webp)
+
+- **Variable Key** — Sets your variable. Use this key to access the variable in other Insights panels.
+- **Type** — Sets the data type of your variable.
+- **Default Value** — Sets a default value for the global variable.
+- **Interface** — Selects the Interface to be used.
+- **Options** — Sets the Interface Options.
 
-Teams that need highly advanced or specialized analytics options are able to extend, customize and modify this module as
-needed. Here are some great resources to get started down that track.
+This panel allows you to create a variable which is globally accessible across other panels. You can use this to do
+things like create a string or number to prefix other Insights Panel titles or pass it as a value into a Filter to query
+results.
 
-- [Extensions > Modules](/extensions/modules)
-- [Extentions > Panels](/extensions/panels)
+To create and assign a Global Variable, follow the steps from [How it Works](#how-it-works), selecting the Global
+Variable panel. Wrap the variable key in double-moustache syntax to access and assign it throughout other Insights
+Panels:
+
+```json
+{{ your_variable_key }}
```

Comment:
@rijkvanzanten I don't understand why this was red... Also, when I tried to do `{{your_variable_key}}` in inline, like I just did... VuePress did not register the `inline` syntax and still treated it as an actual vue variable, rendering an empty  codebox like this -> ` `  

If you're not sure, I will just check this out and fix it... in VitePress 😆

Reply:
It's red here, because that's not valid `json`. Removing the `json` at the top removes the red 🙂 

In VuePress (and I believe VitePress), stuff inside the triple `` ` `` character is automagically escaped, whereas inline `{{ }}` are interpolated as Vue variables. You can wrap those in

````
::: v-pre

{{ your_variable_key }}

:::
````

to not have them being used as a variable